src/components/calculator/PointsDisplay.tsx

```diff
@@ -29,49 +35,296 @@ function displayPercent(points: number): string {
+  const budgetAnnualPercentFormatted = fixedPointPrecisionPercent(
+    props.riskBudget / 1000000,
+  )
+  const weekBudgetFormatted = getWeekBudget(props.riskBudget)
+  const budgetConsumptionFormatted = budgetConsumption(
+    props.points,
+    props.riskBudget,
+  )
+
+  const calculationBreakdown = (
+    <>
+      <span
+        className="expandable-header"
+        onClick={() =>
+          setCalculationExplanation(!displayCalculationExplanation)
+        }
+        aria-controls="calculation-explanation"
+        aria-expanded={displayCalculationExplanation}
+      >
+        {displayCalculationExplanation ? (
+          <>
+            <BsChevronDown /> How this was calculated
+          </>
+        ) : (
+          <>
+            <BsChevronRight /> Learn how this was calculated
+          </>
+        )}
+      </span>
+      <Collapse in={displayCalculationExplanation}>
+        <div id="calculation-explanation">
+          <div>
+            <h4>Calculation:</h4>
+            <div id="calculation-breakdown">
+              <code>
+                ({personRiskEachFormatted} Person Risk) x (
+                {activityRiskFormatted} Activity Risk) x (
+                {props.data.personCount} {personCountSuffixFormatted})<br />
+                <strong style={{ fontSize: '1.5em' }}>
+                  = ~{pointsFormatted} microCOVIDs {frequencyFormatted}
+                </strong>{' '}
+                <br />
+                (margin of error: {lowerBoundFormatted} to {upperBoundFormatted}
+                )
+              </code>{' '}
+            </div>
+          </div>
+          <h4>Calculation Steps:</h4>
+          <ol>
+            <li>
+              <strong>
+                Person Risk:{' '}
+                <code>{personRiskEachFormatted}-in-a-million chance</code>
+              </strong>{' '}
+              <br />
+              First, we calculate that each other person in this area has a{' '}
+              <strong>{personRiskEachFormatted}-in-a-million chance</strong>
+              {'  '} of currently having COVID.
+            </li>
+            <li>
+              <strong>
+                Activity Risk: <code>{activityRiskFormatted} chance</code>
+              </strong>
+              <br /> Next, we calcualte the risk of the activity. Assuming 1
+              person at this activity has COVID, then you would have a{' '}
+              <strong>{activityRiskFormatted} chance</strong> of getting COVID.
+              <b>
+                {activityRisk && activityRisk >= MAX_ACTIVITY_RISK
+                  ? ' (NOTE: We have capped this number at the maximum Activity Risk.)'
+                  : ''}
+                {props.data.distance === 'intimate' &&
+                props.data.duration < intimateDurationFloor
+                  ? ' (NOTE: We have applied a minimum Activity Risk for kissing.)'
+                  : ''}
+              </b>
+            </li>
+            <li>
+              <strong>
```

Comment:
We have this pattern a lot: `Strong: code`. I'd like to see this refactored into a function/component. Can be done in a follow-up.